system/bus.c

```diff
@@ -196,7 +197,9 @@ bus_init (Bus *self)
         self
     );
 
-    if (gestures_xiaomi_supported ())
+    if (gestures_i2c_supported ())
```

Comment:
I would put this at the end, so that it can act as a "fallback". Perhaps some devices that can match `gestures_i2c_support()` might want to implement their own interface anyway to add support for some other device-specific features (like "glove mode" or so on)